File: examples/python/pagination/snapshot.py
from daytona import Daytona


def main():
    daytona = Daytona()

    result = daytona.snapshot.list(page=2, limit=10)
    for snapshot in result.items:
        print(f"{snapshot.name} ({snapshot.image_name})")


if __name__ == "__main__":
    main()

File: examples/typescript/pagination/snapshot.ts
import { Daytona } from '@daytonaio/sdk'

async function main() {
  const daytona = new Daytona()

  const result = await daytona.snapshot.list(2, 10)
  console.log(`Found ${result.total} snapshots`)
  result.items.forEach((snapshot) => console.log(`${snapshot.name} (${snapshot.imageName})`))
}

main().catch(console.error)
